### PR TITLE
Fixissue6

### PR DESCRIPTION
**Pull request by GitHubUser5136**

### CODE REVIEW

File: src/sample/signUpController.java [hunk 1]
```diff
@@ -108,17 +108,18 @@ public void signUp(MouseEvent event) throws IOException {
...
+              writer1.write("\n" + "--Select--");
+              writer1.write("\n" + "--Select--");
+              writer1.write("\n" + "--Select--");
+              writer1.write("\n" + "--Select--");
+              writer1.write("\n" + "--Select--");
+              writer1.write("\n" + "--Select--");
+              writer1.write("\n" + "--Select--");
+              writer1.write("\n" + "--Select--");
+              writer1.write("\n" + "--Select--");
+              writer1.write("\n\n\n\n");
```
Comment (Michelle-Lohwt) [inline src/sample/signUpController.java]: wait wait, I think your version is still old one

**Review by Michelle-Lohwt — CHANGES_REQUESTED** [review]
wait wait, I think your version is still old one

File: src/sample/signUpController.java [hunk 2]
```diff
@@ -119,6 +119,7 @@ public void signUp(MouseEvent event) throws IOException {
               writer1.write("\n" + "--Select--");
               writer1.write("\n" + "--Select--");
               writer1.write("\n" + "--Select--");
+              writer1.write("\n\n\n");
```
Comment (Michelle-Lohwt) [inline src/sample/signUpController.java:122]: why "\n\n\n"?

**Review by ziqiang2344 — APPROVED** [review]

File: src/sample/signUpController.java (hunk 2 above)
Comment (GitHubUser5136) [reply src/sample/signUpController.java:122]: To avoid out of bound error, if not it will cause out of bound error

**Review by Michelle-Lohwt — APPROVED** [review]

**Pull request merged**